PopPUNK/lineages.py

```diff
@@ -0,0 +1,470 @@
+#!/usr/bin/env python
+# vim: set fileencoding=<utf-8> :
+# Copyright 2018-2020 John Lees and Nick Croucher
+
+import os
+import sys
+import argparse
+import subprocess
+import pickle
+import pandas as pd
+from collections import defaultdict
+
+import pp_sketchlib
+from .assign import assign_query, assign_query_hdf5
+from .network import construct_network_from_edge_list, printClusters, save_network
+from .models import LineageFit
+from .plot import writeClusterCsv
+from .sketchlib import readDBParams
+from .qc import prune_distance_matrix, sketchlibAssemblyQC
+from .utils import createOverallLineage, readPickle, setupDBFuncs
```

Comment:
Check all of these are used. pp_sketchlib doesn't seem to be at least

Reply:
Reviewed in 56cc4b0.